### PR TITLE
Acid blob is R.I.P IN ACID

### PR DESCRIPTION
**Pull request by Iamgoofball**
;_;7

Also fixes a bug with LDM and Sorium blobs

### CODE REVIEW

**Comment by Anonus** [comment]
Rip acid blob. Actually no fuck acid blob, let it rest without peace.

**Comment by GunHog** [comment]
Sadness :(

**Comment by RemieRichards** [comment]
i cri

**Comment by Kromgar** [comment]
NEAR FAR. WHEREVER YOU ARE.

File: code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm [hunk 1]
```diff
@@ -187,7 +173,7 @@ datum/reagent/blob/spacedrugs/reaction_mob(var/mob/living/M as mob, var/method=T
 
 /proc/reagent_vortex(var/mob/living/M as mob, var/setting_type)
 	var/turf/pull = get_turf(M)
-	for(var/atom/movable/X in orange(4,pull))
+	for(var/atom/movable/X in range(4,pull))
```
Comment (Supermichael777) [inline code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm]: OOOOOPPPPPSSS.

Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm]: this is intentional super

orange() = everything but the tile it was on, making sorium blob only throw everything but the guy you hit

range() = everything in that radius

**Pull request merged**